core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/RegionPlanGenerator.scala

```diff
@@ -130,7 +130,6 @@ abstract class RegionPlanGenerator(
 
     var newPhysicalPlan = physicalPlan
       .removeLink(physicalLink)
-      .setOperatorUnblockPort(toOp.id, toPortId)
```

Comment:
I agree we need to remove this line. but please verify if you remove this line, can RPGs still work properly for dependent ports? I have a feeling that if we remove this line, we need to write an alternative logic, but I do not find the alternative logic.

Reply:
not needed